src/v2/Apps/Order/Components/SavedAddresses.tsx

```diff
@@ -0,0 +1,145 @@
+import { RadioGroup, BorderedRadio, Spacer, Flex, Text } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+import { SavedAddresses_me } from "v2/__generated__/SavedAddresses_me.graphql"
+
+const EditButton = styled(Text)`
+  &:hover {
+    text-decoration: underline;
+  }
+`
+
+const renderAddressList = (addressList, handleClickEdit) => {
```

Comment:
Sinice this returns JSX, maybe it can just be a conventional react component `<AddressList />` that accepts two props?